aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-7.md

```diff
@@ -37,31 +37,31 @@ Change the namespace for the Controller and Model classes so they begin with Mvc
 
 The updated solution should look like the following:
 
-![](mvc-music-store-part-7/_static/image1.png)
+![Screenshot of the Solution Explorer pane with AccountController.cs, AccountModels.cs, and the files in the /Views/Account/ folder highlighted in red.](mvc-music-store-part-7/_static/image1.png)
```

Comment:
Write as "Account Controller dot C S, Account Models dot CS, and the files in the / Views / Account / folder..."